### PR TITLE
Change copy model stage in 208 notebook

### PR DESCRIPTION
**Pull request by sbalandi**

### CODE REVIEW

**Comment by eaidova** [comment]
```
TypeError: copytree() got an unexpected keyword argument 'dirs_exist_ok'
```
looks like you use api introduced in python3.8, all builds for python3.7 failed

File: notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb [hunk 1]
```diff
@@ -166,49 +166,19 @@
...
-    "    for model_file in omz_model_dir.iterdir():\n",
-    "        try:\n",
-    "            shutil.copyfile(model_file, model_dir / model_file.name)\n",
-    "        except FileExistsError:\n",
-    "            pass\n",
+    "# detection_model_info subdirectory\n",
+    "model_dir.mkdir(exist_ok=True)\n",
+    "\n",
+    "# detection_model_info subdirectory and recognition_model_info subdirectory\n",
+    "omz_model_dirs = [base_model_dir / Path(\"public/text-recognition-resnet-fc\") / precision,\n",
```
Comment (eaidova) [inline notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:173]:
```suggestion
    "omz_model_dirs = [base_model_dir / \"public/text-recognition-resnet-fc\" / precision,\n",
```

File: notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb [hunk 2]
```diff
@@ -166,49 +166,19 @@
...
+    "                  base_model_dir / Path(\"intel/horizontal-text-detection-0001\") / precision]\n",
```
Comment (eaidova) [inline notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:174]:
```suggestion
    "                  base_model_dir / \"intel/horizontal-text-detection-0001\" / precision]\n",
```

File: notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb (hunk 1 above)
Comment (sbalandi) [reply notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb]: done

File: notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb (hunk 2 above)
Comment (sbalandi) [reply notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb]: done

File: notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb [hunk 3]
```diff
@@ -166,49 +166,19 @@
...
+    "\n",
+    "# detection_model_info subdirectory and recognition_model_info subdirectory\n",
+    "omz_model_dirs = [base_model_dir / \"public/text-recognition-resnet-fc\" / precision,\n",
+    "                  base_model_dir / \"intel/horizontal-text-detection-0001\" / precision]\n",
+    "\n",
+    "for omz_model_dir in omz_model_dirs:\n",
+    "    print(f'from {omz_model_dir} to {model_dir} copied:')\n",
+    "    for file_path in omz_model_dir.iterdir():\n",
+    "        if file_path.is_file():\n",
+    "            shutil.copy2(file_path, model_dir)\n",
```
Comment (adrianboguszewski) [inline notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:180]: I know this is not work done in this PR, but I'm not a fan of downloading anything to some external directory like "~/open_model_zoo_models" and then copying it. I think it creates more mess (more directories containing files related to one case) and unnecessary code. Could we fix it for 208 in this PR? Let's download everything to a local "model" dir.

Comment (sbalandi) [reply notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb]: yes, done

**Comment by eaidova** [comment]
@sbalandi looks like last changes requires update ci scripts:
```
   File "/tmp/scripts/patch_notebooks.py", line 37, in patch_notebooks
    raise ValueError(
ValueError: Processing /opt/app-root/notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb failed: ~/open_model_zoo_cache does not exist in cell
```
and code style:
```
notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:cell_1:1:1: F401 'shutil' imported but unused
```

**Review by adrianboguszewski — APPROVED** [review]
Good job!

**Review by adrianboguszewski — COMMENTED** [review]
One more ask. Could you also include a fix for this? #255

**Comment by sbalandi** [comment]
> @sbalandi looks like last changes requires update ci scripts:
> 
> ```
>    File "/tmp/scripts/patch_notebooks.py", line 37, in patch_notebooks
>     raise ValueError(
> ValueError: Processing /opt/app-root/notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb failed: ~/open_model_zoo_cache does not exist in cell
> ```
> 
> and code style:
> 
> ```
> notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb:cell_1:1:1: F401 'shutil' imported but unused
> ```

thank you ! fixed

**Pull request merged**